contracts/test/tokens/BurnableTestToken.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier:MIT
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20Burnable.sol";
+
+contract BurnableTestToken is ERC20("Test Token", "TKN") {
+    function mint(uint amount, address to) public {
+        _mint(msg.sender, amount);
+        _transfer(msg.sender, to, amount);
+    }
+
+    function burn(uint amount) public {
+        _burn(msg.sender, amount);
+    }
+}
```

Comment:
Please tell me where are you using this. I couldn't see it in use.

Reply:
You are right, It's a leftover and I will remove it.